### DISCUSSION TITLE
The commit status

### DISCUSSION THREAD

**Discussion by satvik-1203** [community/community#5121] [Mobile] [answered]
How about adding commit status in the home screen?

**Comment by brianlovin** [comment, marked as answer]
Do you mean that you'd like to see this on each repository list item in the Favorites section?

**Reply by satvik-1203** [reply]
I didn't quite get you, but what I meant was adding the commit status bar which shows all your contributions and communities you contributed to(similar to the home screen commit status bar that is on the web page) on mobile as well.

**Reply by brianlovin** [reply]
Ah, are you referring to this? 

<img width="766" alt="Screen Shot 2021-10-11 at 18 00 44@2x" src="https://user-images.githubusercontent.com/1923260/136873030-157270eb-f256-4f19-a6b2-0a2949a01075.png">

**Reply by satvik-1203** [reply]
Yeah that :)

**Reply by brianlovin** [reply]
Ah, gotcha. On iOS we do support that contribution graph as a widget that you can add to your phone's Home Screen. We haven't added support for showing this on user profiles yet, but will hopefully be able to add that soon 😊

**Reply by satvik-1203** [reply]
Yeah I observed that haha, Thanks for considering it :)
